plaidml/BUILD

```diff
@@ -159,6 +159,8 @@ plaidml_cc_test(
         ":api",
         "//testing:matchers",
         "//testing:plaidml_config",
+        "@tbb",
+        "@xsmm",
     ],
```

Comment:
Why are we adding these deps here? Seems like they should be transitively depended on. Basically, I don't see how the `uint8_test` is directly dependent on `@tbb` or `@xsmm`

Reply:
Yes... This is left over from testing. Taking it out.